components/modal/config/ModalOptions.cs

```diff
@@ -134,6 +134,12 @@ public bool ConfirmLoading
         /// </summary>
         public RenderFragment RestoreBtnIcon { get; set; } = DialogOptions.DefaultRestoreIcon;
 
+        /// <summary>
+        /// Maximize the Modal during component initialization, and it will ignore the Maximizable value.
+        /// </summary>
+        [Parameter]
+        public bool MaximizationOnInit { get; set; } = false;
```

Comment:
It would be better to rename it `Maximized` instead, and user can use this parameter to toggle the maximization. Like visible.

Reply:
Sorry, this may cause confusion of component status. Because one of them belongs to the property of the component and the other belongs to the state of the component. 

Like Visible, it cannot be modified within a component, but can only be passed from as a property.